media/img/home/voices/promos/webmaker/webmaker-teacher.svg

```diff
@@ -0,0 +1,3 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" [
+	<!ENTITY ns_flows "http://ns.adobe.com/Flows/1.0/">
```

Comment:
Huh, surprised the optimizer left this in... it certainly seems unnecessary. I'll update.